templates/layouts/main.html

```diff
@@ -45,7 +45,7 @@
 
   <header class="site-header">
     <div class="disclaimer">
-      <span class="disclaimer__left">Looking for <a href="http://www.fec.gov">fec.gov?</a></span>
+      <span class="disclaimer__left">Looking for <a href="http://www.fec.gov">FEC.gov?</a></span>
```

Comment:
Excellent note. And so glad you're reading our content guide!

We've been using `FEC.gov` and `beta.FEC.gov` in our other writings. But you're correct that it isn't aligned with our (well reasoned, well supported)18F style of always using lowercase. 

We *can* switch to lowercase (for consistency), but I think we'll need to change all our references to `fec.gov` or `beta.fec.gov`. 

@leahbannon — thoughts on this?